platform/ui/src/components/Select/Select.jsx

```diff
@@ -1,61 +1,115 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
-import ReactSelect from 'react-select';
+import ReactSelect, { components } from 'react-select';
 
 import './Select.css';
 
+const MultiValueContainer = ({ data }) => {
+  return <span>{`${data.label}, `}</span>;
+};
+
+const MultiValue = props => {
+  return (
+    <components.MultiValue {...props}>
+      <span>{props.data.label}</span>
+    </components.MultiValue>
+  );
+};
+
+const Option = props => {
+  return (
+    <div>
+      <components.Option {...props}>
+        <input
+          type="checkbox"
+          checked={props.isSelected}
+          className="w-6 h-6"
+          onChange={e => null}
+        />{' '}
+        <label>{props.value} </label>
+      </components.Option>
+    </div>
+  );
+};
+
 const Select = ({
-  autoFocus,
   className,
+  closeMenuOnSelect,
+  hideSelectedOptions,
+  isClearable,
   isDisabled,
   isMulti,
   isSearchable,
-  name,
   onChange,
   options,
   placeholder,
-  noOptionsMessage,
   value,
 }) => {
+  const _components = isMulti
+    ? { MultiValueContainer, Option, MultiValue }
+    : {};
+
   return (
     <ReactSelect
-      autoFocus={autoFocus}
       className={classnames(
         className,
-        'flex flex-col flex-1 mt-2 customSelect__wrapper'
+        'flex flex-col flex-1 customSelect__wrapper'
```

Comment:
Maybe we could define a `min-width` for this component, here's how it's being displayed in the docz page for Select:

![image](https://user-images.githubusercontent.com/1905961/77667445-f8cdc580-6f60-11ea-93eb-0fb85797345d.png)

I know it depends on the parent width, but to avoid break lines like this, we could just set a `min-width`.

What do you think?

Reply:
I haven't touch the react-select css for that. I can do it thought. 
Ideally it works like this so it can expand and always show all selected itens. 
I don't think setting a min with will solve this. cause it will depends on the size of the options as well. 

We would need to think on how to deal with bigger inputs, as we did for the InputMultiSelect, when we wrap the content and add ellipses. But this was an especific scenario.

Reply:
Ok, let's only add a bigger width in the doc page as suggested